llvm/docs/GitHub.rst

```diff
@@ -131,11 +131,45 @@ You can also merge via the CLI by switching to your branch locally and run:
 
   gh pr merge --squash --delete-branch
 
+If you observe an error message from the above informing you that your pull
+request is not mergeable, then that is likely because upstream has been
+modified since your pull request was authored in a way that now results in a
+merge conflict. You must first resolve this merge conflict in order to merge
+your pull request. In order to do that:
+
+::
+
+  git fetch origin
```

Comment:
This should probably be upstream?

Reply:
ah, right; my remotes aren't named that but to be consistent with the rest of the doc, you're right.

I may have messed something up when I started using `gh` recently; IIRC it was supposed to rename my remotes?

Reply:
done in 5f8e1064311a